test/functional/services/lib/web_element_wrapper/web_element_wrapper.ts

```diff
@@ -218,12 +259,14 @@ export class WebElementWrapper {
   public async pressKeys<T extends IKey>(keys: T | T[]): Promise<void>;
   public async pressKeys<T extends string>(keys: T | T[]): Promise<void>;
   public async pressKeys(keys: string): Promise<void> {
-    if (Array.isArray(keys)) {
-      const chord = this.Keys.chord(keys);
-      await this._webElement.sendKeys(chord);
-    } else {
-      await this._webElement.sendKeys(keys);
-    }
+    await this.retryCall(async function pressKeys(wrapper: WebElementWrapper) {
+      if (Array.isArray(keys)) {
+        const chord = wrapper.Keys.chord(keys);
+        await wrapper._webElement.sendKeys(chord);
+      } else {
+        await wrapper._webElement.sendKeys(keys);
+      }
+    });
```

Comment:
*Could be*  _off the top of my head_
```
const send = async wrapper => wrapper._webElement.sendKeys.bind(wrapper)
const press = keys => async wrapper => [].isArray(keys) ? await send(wrapper.Keys.chord(keys)) : await send(keys)
const pressKeys = press(keys)
await this.retryCall(pressKeys())
```
*Half the lines of code*

Reply:
I personally find the existing code a lot easier to read.